VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
-              {personalInfo?.fullName}
-            </TextView>
-            {accessToMilitaryInfo && (
-              <Box display="flex" flexDirection="row">
-                {getBranchSeal()}
-                <TextView ml={10} variant="MobileBody" color="primaryContrast" testID="veteranStatusBranchTestID">
-                  {branch}
-                </TextView>
-              </Box>
-            )}
-          </Box>
-        </Box>
-        <Box mx={theme.dimensions.gutter}>
-          {ratingIsDefined && (
-            <Box {...boxProps}>
-              <TextView variant="MobileBodyBold" color="primaryContrast">
-                {t('disabilityRating.title')}
-              </TextView>
-              <TextView variant="MobileBody" color="primaryContrast" testID="veteranStatusDisabilityRatingTestID">
-                {combinedPercentText}
+    <LargePanel title={t('veteranStatus.title')} rightButtonText={t('close')} dividerMarginBypass={true} removeInsets={true} testID="veteranStatusTestID">
+      <Box mx={isPortrait ? theme.dimensions.gutter : theme.dimensions.headerHeight} alignItems="center" mt={theme.dimensions.standardMarginBetween}>
+        <VAIcon name={'Logo'} />
+        {/* <Box my={theme.dimensions.standardMarginBetween}>
+        //TODO: Put back PhotoUpload later after concerns have been met
+          <PhotoUpload width={100} height={100} />
+        </Box> */}
+        <Box my={theme.dimensions.formMarginBetween}>
+          <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast">
+            {personalInfo?.fullName}
+          </TextView>
+          {accessToMilitaryInfo && (
+            <Box display="flex" flexDirection="row">
+              {getBranchSeal()}
+              <TextView ml={10} variant="MobileBody" color="primaryContrast">
+                {branch}
               </TextView>
             </Box>
           )}
+        </Box>
+      </Box>
+      <Box mx={isPortrait ? theme.dimensions.gutter : theme.dimensions.headerHeight}>
+        {ratingIsDefined && (
           <Box {...boxProps}>
             <TextView variant="MobileBodyBold" color="primaryContrast">
-              {t('veteranStatus.periodOfService')}
-            </TextView>
-            {getPeriodOfService}
-          </Box>
-          <Box {...boxProps} borderBottomWidth={theme.dimensions.borderWidth} mb={theme.dimensions.formMarginBetween}>
-            <TextView variant="MobileBodyBold" color="primaryContrast">
-              {t('personalInformation.dateOfBirth')}
+              {t('disabilityRating.title')}
             </TextView>
-            <TextView variant="MobileBody" color="primaryContrast" testID="veteranStatusDOBTestID">
-              {personalInfo?.birthDate || t('personalInformation.informationNotAvailable')}
+            <TextView variant="MobileBody" color="primaryContrast">
```

Comment:
removed testid?

Reply:
good catch, the merge conflict with Rachel's was a tricky one.